### DISCUSSION TITLE
Finding also in address?

### DISCUSSION THREAD

**Discussion by Petr-Ledvina** [tomik23/Leaflet.Autocomplete#29] [Ideas]
Hi, would it be possible to add a parameter to your great app to search by address (e.g. street, house number), not just by city? That way the Nominatim geolocation would have only a slightly different format. Thanks in advance!

**Comment by tomickigrzegorz** [comment]
You can add it yourself, [here is detailed documentation](https://nominatim.org/release-docs/develop/) on how to do the search, I recommend reading it.

PS.
How would you recognize a given string as a city or a street or a house number? There are countless combinations, how would you build such a URL using one field? I'm curious about your solution.

**Comment by Petr-Ledvina** [comment]
E.g. "Enter the adress" in field. And than e.g.: https://nominatim.openstreetmap.org/search?format=json&accept-language=cs&q=Hlinky%20510%20Brno  thus through the parameter q. Like another solution https://github.com/k4r573n/leaflet-control-osm-geocoder  I know I could modify the script, but extended search (parameterization) seems like a more elegant solution.

**Reply by tomickigrzegorz** [reply]
Well, all you have to do is change this line https://github.com/tomickigrzegorz/Leaflet.Autocomplete/blob/master/docs/script.js#L9
Instead of "city" use "q"
